backend/src/main/java/com/greenfoxacademy/backend/dtos/RegisterUserDTO.java

```diff
@@ -0,0 +1,18 @@
+package com.greenfoxacademy.backend.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * Data Transfer Object for User Entity.
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class RegisterUserDto {
```

Comment:
Can you change the name of the class?

Reply:
I have now changed it on github as my push didn't carry the refactoring..
![image](https://github.com/gfa-cc-after/Pet-Clinic-GilmoreDevs/assets/149159395/8d8f1959-56c0-4fdc-a1c9-025d172d0b6d)